posts/2019-04-26-Sunsetting-IRC.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Sunsetting IRC"
+author: The Rust Core Team
+release: true
+---
+
+The Rust community has had a presence on Mozilla’s IRC network almost since Rust’s inception. Over time, the single channel grew into a set of pretty active channels where folks would come to ask Rust questions, coordinate work on Rust itself, and just in general chat about Rust.
+
+Mozilla [recently announced][mhoye-post] that it would be shutting down its IRC network, citing a growing maintenance and moderation burden. They are looking into new options for the Mozilla community, but this does leave the question open as to what the Rust project will do.
+
+Last year a lot of the teams [started exploring new communication platforms][new-platforms]. Almost all the Rust teams no longer use IRC as their official discussion platform, instead using Discord or Zulip (as well as a variety of video chat tools for synchronous meetings). The few teams that do use IRC are working with us to find a new home, likely a channel on Discord or Zulip.
+
+This leaves the #rust and #rust-beginners channels on Mozilla’s IRC network, which are still quite active, that will need a new home when Mozilla’s network shuts down. Rust’s [official Discord server][discord] does have the #users, #help, and #beginners channels that fill in this purpose, and we recommend people start using those.
+
+We understand that not everyone wishes to switch to Discord for many reasons. For people who wish to continue using IRC, there is [an unofficial freenode channel][freenode] which you can hang out in, though we’d like to emphasize that this is not associated with the Rust teams and is not moderated by our Moderation team. You’re also free to create new channels on freenode [in accordance with the freenode rules][freenode-rules].
+
+There are still a couple months before irc.mozilla.org shuts down — we’ll work at making this transition as smooth as possible in this time. We’d like to thank everyone who made #rust and #rust-beginners on Mozilla IRC a great place to hang out and are sad to see it go.
```

Comment:
```suggestion
There are still a couple months before irc.mozilla.org shuts down — we’ll work at making this transition as smooth as possible in this time. We’d like to thank everyone who made #rust and #rust-beginners on Mozilla IRC a great place to hang out! We are sad to see it go. 😢 
```

The structure of this sentence tripped me up on first read, so I'm trying to simplify it. Also, I like emojis. =)

Reply:
Yeah it was awkward to me too, but I wasn't sure how to fix it. The emoji helps!

Reply:
oops, got overwritten in the force push, but I pushed up a fix